install/0000_80_machine-config-operator_01_machineconfigpool.crd.yaml

```diff
@@ -69,6 +69,10 @@ spec:
     # shortNames allow shorter string to match your resource on the CLI
     shortNames:
     - mcp
+  template:
+    metadata:
+      labels:
+          pools.operator.machineconfiguration.openshift.io/{{ status.configuration.name }}: ""
```

Comment:
omg niceee

Reply:
> omg niceee

hmm..... What do you think this does?

Reply:
Having a default label applied to each MCP object being created, do you know what the correct way to do this is?